client/src/pages/CreateOrganizationProfile.js

```diff
@@ -156,7 +154,7 @@ const CreateOrgProfile = (props) => {
             <Input
               type="text"
               required
-              placeholder="notion"
+              placeholder=""
```

Comment:
Perhaps we should add a different place holder text instead of having it empty @vikjoshi ?

Reply:
@idjevm , the field is "Organization Name" which is pretty self-explanatory so my thinking is that we don't need any placeholder text there

Reply:
That makes sense, then I would probably suggest to keep consistency and also remove the place holder for the email the `example@example.com`, and use email validation util to make sure that the user inputs a valid email, the validate email util also display an error message I believe.

Reply:
@vikjoshi Let us know about my last comment ^.

Reply:
@idjevm agree with your last comment- let's go ahead and remove the placeholder text for email and use error handling/email validation for error messages instead. Thank you!